comment/tests/test_mixins.py

```diff
@@ -223,14 +271,78 @@ def setUp(self):
     def test_user_cannot_subscribe(self):
         self.client.force_login(self.user_1)
         self.assertFalse(settings.COMMENT_ALLOW_SUBSCRIPTION)
+
         response = self.client.post(self.toggle_follow_url, HTTP_X_REQUESTED_WITH='XMLHttpRequest')
 
         self.assertEqual(response.status_code, 403)
+        self.assert_permission_denied_response(response, reason=FollowError.SYSTEM_NOT_ENABLED)
 
     @patch.object(settings, 'COMMENT_ALLOW_SUBSCRIPTION', True)
     def test_user_can_subscribe(self):
         self.client.force_login(self.user_1)
         self.assertTrue(settings.COMMENT_ALLOW_SUBSCRIPTION)
+
         response = self.client.post(self.toggle_follow_url, HTTP_X_REQUESTED_WITH='XMLHttpRequest')
 
         self.assertEqual(response.status_code, 200)
+
+
+class CanBlockUsersMixinTest(BaseCommentMixinTest):
+    def setUp(self):
+        super().setUp()
+        self.toggle_blocking_url = self.get_url(reverse('comment:toggle-blocking'))
+
+    @patch.object(settings, 'COMMENT_ALLOW_BLOCKING_USERS', False)
+    def test_admin_cannot_block_users_when_system_disabled(self):
+        self.client.force_login(self.admin)
+        self.assertFalse(settings.COMMENT_ALLOW_BLOCKING_USERS)
+        data = {'comment_id': self.comment.id}
+
+        response = self.client.post(self.toggle_blocking_url, data=data, HTTP_X_REQUESTED_WITH='XMLHttpRequest')
```

Comment:
I think we can override the default client with the one request this ajax header in the `setUpTestData`  function for the class`BaseCommentMixinTest`. We have done the same in `BaseCommentViewTest` here: https://github.com/Radi85/Comment/blob/develop/comment/tests/base.py#L261